prometheus/prometheus-config.yml

```
@@ -0,0 +1,7 @@
+global:
+  scrape_interval: 15s
+
+scrape_configs:
+  - job_name: 'peer-observer-metrics'
+    static_configs:
+      - targets: ['metrics:8001']
```

To allow for a multi-node setup, I use the label `host` in my set up for the dashboards. Even if you just have one host, you probably want to add a name for it here. See also https://github.com/0xB10C/peer-observer/tree/master/tools/metrics#grafana-dashboards
```suggestion
      - targets: ['metrics:8001']
        labels:
          host: node1
```